Makefile

```diff
@@ -57,7 +66,7 @@ dev-server: checks
 	@echo "|                                                |"
 	@echo "+------------------------------------------------+$(RESET)\n\n"
 
-	$(MAKE) desktop$/config.json CONFIG_ENV=$(CONFIG_ENV)
+	$(MAKE) desktop/config.json CONFIG_ENV=$(CONFIG_ENV)
```

Comment:
was this change intentional? the `$/` is a guard for cross-platform file references

Reply:
Yep - this is very intentional. 

On Windows, it's common for devs to use Git bash. Git bash (derived from MSYS2) expects *nix-style paths, and invoking `make desktop\config.json` _won't_ work (because of how MSYS  attempts to translate file paths). Seems coding it this way was assumed to work, and apparently wasn't tested because it actually doesn't. 

One of the key advantages of using MSYS/Git/Windows bash is, ideally, we should be able to use forward-slashes and `:` path separators and let the environment handle the paths accordingly (there are exceptions which we can mark for exclusion, if/where necessary).

TL;DR I'd rather avoid as much platform-specific exceptions like this, and I've confirmed that this works just fine (i.e. you can invoke `make desktop/config.json` on Windows and it will do the right thing (i.e. execute based on whether or not the path exists).

Reply:
> Seems coding it this way was assumed to work, and apparently wasn't tested because it actually doesn't.

that's not entirely true. it was broken, then this was added, then it worked, now it's apparently broken again in yet-another Windows environment 😉 

if we're going to remove it then maybe we should remove it everywhere, and probably in a precursor PR

Reply:
Also worth noting that the removal of `$/` is limited to this target of the Makefile. It's usage may/may not be necessary in other existing targets, but that's TBD and not in scope for the time being.

Reply:
oh okay. well, things may break. if they do we'll fix them. carry on

Reply:
> ... it was broken, then this was added, then it worked, now it's apparently broken again in yet-another Windows environment

Yeah. I'm beginning to learn Windows can be ... funny that way. 🤔 

Interestingly enough this error was consistent in AppVeyor as well, which also uses MSYS2 - that could be the common denominator.